### ISSUE TITLE
Standard Orbeon error dialog not read out

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#1724]
E.g. the session expired dialog should have appropriate ARIA attribute.

Set the ARIA [dialog](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_dialog_role) role.

[Customer thread](https://basecamp.com/1721271/projects/3641011/messages/21619946#comment_157363156)
